files/es/web/javascript/reference/global_objects/array/reverse/index.html

```diff
@@ -17,7 +17,7 @@
 
 <h2 id="Sintaxis" name="Sintaxis">Sintaxis</h2>
 
-<pre class="syntaxbox"><var>a</var>.reverse()</pre>
+<pre class="syntaxbox"><var>arr</var>.reverse()</pre>
```

Comment:
De hecho, ya que estamos hablando de `reverse()` solo se incluye la palabra reservada.
y tambien la documentacion de `content` lo indica y refuerza.

https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/array/reverse/index.md?plain=1#L21

```suggestion
<pre class="syntaxbox">reverse()</pre>
```